### PR TITLE
Azure: Add Deployment engine docs

### PR DESCRIPTION
**Pull request by ariordan-redhat**
Azure:
Describe how to open the deployment engine, and use it to monitor and cancel AAP deployments.

### CODE REVIEW

**Comment by bhavenst** [comment]
Looks good!  I added some comments to the Google doc draft rather than here before I realized I could do it here.  Hope that is OK!

File: stories/topics/proc-azure-cancel-deployment.adoc [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+. Login to the deployment engine to display the progress of the deployment steps in the *Ansible Automation Platform Deployment Engine* page.
+Refer to xref:azure-monitor-deployment-engine_aap-azure-install[Monitoring deployments on the {PlatformNameShort} Deployment Engine] for information on accessing and logging into the *Ansible Automation Platform Deployment Engine* page.
+
+. To cancel the deployment, click btn:[Cancel Deployment] and confirm.
++
+This action cancels all the remaining steps in the deployment, including the currently running step. It also cancels pending steps in the deployment.
++
+The status for the steps that have not been executed updates to *Cancelled*.
+To view the deployment processes on Azure, open the *Overview* page for your deployment on the Azure web interface and select *Deployments*.
```
Comment (qiwangrh) [inline stories/topics/proc-azure-cancel-deployment.adoc:18]: I think this needs to be changed to "To view the deployment processes on Azure, open the "Overview page" for your managed resource group in which you deployed the Ansible Automation Platform on the Azure web interface and select "Deployments".

File: stories/topics/proc-azure-cancel-deployment.adoc [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
++
+[IMPORTANT]
+====
+Cancelling the deployment does not delete the managed application from your Azure subscription.
+To avoid incurring costs for the managed application and other resources that are still running, you must delete them.
+====
+. To delete Azure resources, navigate to the resource group for your deployment in the Azure portal. Select the resources you want to delete and click btn:[Delete].
+For more information about deleting resources, refer to link:https://learn.microsoft.com/en-us/azure/azure-resource-manager/management/manage-resources-portal[Manage Azure resources by using the Azure portal] in the Microsoft Azure documentation.
+
+You cannot restart a deployment after you have canceled it.
```
Comment (qiwangrh) [inline stories/topics/proc-azure-cancel-deployment.adoc:28]: What does "restart a deployment" here mean? After customers cancel a deployment and delete all the cancelled deployment related resources from Azure, they can kick off a new deployment if they want.

File: stories/topics/proc-azure-monitor-deployment-engine.adoc [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+[role="_abstract"]
+
+The deployment engine displays information about your {AAPonAzureNameShort} deployment.
+You can monitor the progress of the deployment and you can cancel the deployment.
+
+When you begin deploying {AAPonAzureNameShort}, the Azure interface displays the *Overview* page for the deployment.
+
+. Click *Parameters and Outputs* in the *Settings* menu.
++
+Approximately 10 minutes into the deployment process, the *Outputs* section of the *Parameters and Outputs* page displays a link to the *`deploymentEngineUrl`*.
```
Comment (qiwangrh) [inline stories/topics/proc-azure-monitor-deployment-engine.adoc:14]: The "Outputs" section shows nothing. If we want to find the Deployment Engine URL, we need to go to the "Parameters and Outputs" section of the deployed managed application after the Azure web shows "Your deployment is complete".

Comment (bhavenst) [reply stories/topics/proc-azure-monitor-deployment-engine.adoc:14]: I think she is referring to the outputs portion of the Parameters and Outputs page, where it says 4 outputs in this screenshot.  So it seems accurate to me.
image

File: stories/topics/proc-azure-monitor-deployment-engine.adoc [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+. Copy the link and paste it in another browser tab to open the login page for the deployment engine.
+. Login to the deployment using the following credentials:
+  * *Username*: _admin_
+  * *Password*: Use the _Administrator Password_ that you chose when configuring your deployment.
+
+The {PlatformNameShort} Deployment Engine displays a list of the steps in the deployment process. Icons indicate the steps that have been completed and the steps that are in progress.
```
Comment (qiwangrh) [inline stories/topics/proc-azure-monitor-deployment-engine.adoc:20]: Icons also indicate the deployment steps which are not started to run yet.

File: stories/topics/proc-azure-provisioning-aap.adoc [hunk 5]
```diff
@@ -38,6 +38,10 @@ Keep this resource group isolated from other resource groups, including the _Res
 
 The application will begin provisioning.
 
+You can use the deployment engine to view the progress of your deployment approximately 10 minutes after you begin provisioning.
```
Comment (qiwangrh) [inline stories/topics/proc-azure-provisioning-aap.adoc:41]: I prefer to say "You can use the deployment engine to view the progress of your deployment approximately a few minutes after Azure web shows "Your deployment is complete". During my testing, if i tried to access the Deployment Engine UI immediately after Azure web shows "Your deployment is complete", I would get "the site is not accessible". The deployment engine URL was becoming accessible after I waited for a few minutes.

File: stories/topics/proc-azure-monitor-deployment-engine.adoc (hunk 3 above)
Comment (qiwangrh) [reply stories/topics/proc-azure-monitor-deployment-engine.adoc:14]: When I read the document, it linked me to the "Output" section on this screen. 
image
 
On this screen, the "Output" was always empty even though Azure Web showed "Your deployment is complete!". @bhavenst , If we want to find the Deployment Engine URL, we need to go to the "Parameters and Outputs" section of the deployed managed application after the Azure web shows "Your deployment is complete". This is the exact screen as what you shared above.

File: stories/topics/proc-azure-cancel-deployment.adoc (hunk 2 above)
Comment (ariordan-redhat) [reply stories/topics/proc-azure-cancel-deployment.adoc:28]: I deleted the sentence. It was mentioned in the demo video.

**Review by ianf77 — APPROVED** [review]
LGTM

**Comment by ianf77** [comment]
Shall I merge?

File: stories/topics/proc-azure-monitor-deployment-engine.adoc (hunk 4 above)
Comment (qiwangrh) [reply stories/topics/proc-azure-monitor-deployment-engine.adoc]: @ariordan-redhat , you are right. There is no icon for steps that have not begun. You can close my comment on it.

**Review by qiwangrh — APPROVED** [review]

**Pull request merged**